### PR TITLE
[Enhance]: update pr template

### PR DESCRIPTION
**Pull request by ZwwWayne**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

This PR updates the PR template.

## Modification

The checklist in the template has been updated and use check mark for better usage.

## BC-breaking (Optional)

N/A

## Use cases (Optional)

N/A

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ If this PR introduces a new feature, it is better to list some use cases here, a
 
 ## Checklist
 
-1. Pre-commit or other linting tools are used to fix the potential lint issues.
-2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
-3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
-4. The documentation has been modified accordingly, like docstring or example tutorials.
+- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
+- [ ] CLA has been signed and all committers have signed the CLA in this PR.
```
Comment (mzr1996) [inline .github/pull_request_template.md:23]: Contributors will be asked to sign CLA after creating the first PR.
If we put it in the checklist, maybe fresh contributors cannot find where to sign it before creating the PR.

**Review by zhouzaida — APPROVED** [review]
LGTM

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -19,7 +19,14 @@ If this PR introduces a new feature, it is better to list some use cases here, a
 
 ## Checklist
 
-1. Pre-commit or other linting tools are used to fix the potential lint issues.
-2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
-3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
-4. The documentation has been modified accordingly, like docstring or example tutorials.
+**Before PR**:
+
+- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
```
Comment (innerlee) [inline .github/pull_request_template.md:24]: Add a link to "what is and how to use pre-commit"?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -19,7 +19,14 @@ If this PR introduces a new feature, it is better to list some use cases here, a
...
+- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
```
Comment (innerlee) [inline .github/pull_request_template.md:25]: ~~fully~~

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -19,7 +19,14 @@ If this PR introduces a new feature, it is better to list some use cases here, a
...
+- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
```
Comment (innerlee) [inline .github/pull_request_template.md:26]: This is for new functionalities. Typos/docs does not need tests. Refactors are covered by existing tests. Bugs are listed int above item

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -19,7 +19,14 @@ If this PR introduces a new feature, it is better to list some use cases here, a
...
+- [ ] The documentation has been modified accordingly, like docstring or example tutorials.
```
Comment (innerlee) [inline .github/pull_request_template.md:27]: like -> including

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -19,7 +19,14 @@ If this PR introduces a new feature, it is better to list some use cases here, a
...
+
+**After PR**:
+
+- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMCls.
```
Comment (innerlee) [inline .github/pull_request_template.md:31]: with those projects -> with some of those projects

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -19,7 +19,15 @@ If this PR introduces a new feature, it is better to list some use cases here, a
 
 ## Checklist
 
-1. Pre-commit or other linting tools are used to fix the potential lint issues.
-2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
-3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
-4. The documentation has been modified accordingly, like docstring or example tutorials.
+**Before PR**:
+
+- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to create this PR.
```
Comment (zhouzaida) [inline .github/pull_request_template.md:24]: should it add a tip for the copyright here

**Review by innerlee — APPROVED** [review]

**Pull request merged**